cosmoz-bottom-bar-view.js

```diff
@@ -102,7 +102,7 @@
 			this._lastScroll = scrollTop;
 		},
 
-		_computeScrollerContentStyle: function (barHeight, bottomBarVisible, fixed) {
+		_computeScrollerContentStyle: function (barHeight, bottomBarVisible) {
```

Comment:
@plequang Did we drop the `fixed` stuff completely?

Reply:
`fixed` has been dropped from `cosmoz-bottom-bar`,  but it's still there in `cosmoz-bottom-bar-view`, when we don't want the bottom-bar to auto-hide when scrolling.